pkg/cvo/sync_worker.go

```diff
@@ -371,47 +377,47 @@ func (w *SyncWorker) syncOnce(ctx context.Context, work *SyncWork, reporter Stat
 	glog.V(4).Infof("Running sync %s", versionString(work.Desired))
 	update := work.Desired
 
-	// cache the image until the release image changes
-	image := w.image
-	if image == nil || !equalUpdate(configv1.Update{Image: image.ReleaseImage}, update) {
-		glog.V(4).Infof("Loading image")
+	// cache the payload until the release image changes
+	validPayload := w.payload
```

Comment:
nit: this change `s/image/validPayload/` seems to reverse the idea from https://github.com/openshift/cluster-version-operator/pull/98

Reply:
nvm, i think it is to keep up with package name `pkg/payload`.

Reply:
Yeah, I did a find and replace before, but went back to being it here to have payload be the internals.